cogs/slashsync/slashsync.py

```diff
@@ -0,0 +1,38 @@
+"""SlashSync
+
+Sync slash commands from the bot to the server.
+"""
+import logging
+
+from redbot.core import checks, commands
+from redbot.core.commands.context import Context
+from redbot.core.utils.chat_formatting import info, success
+from redbot.core.bot import Red
```

Comment:
import sorting: should `redbot.core.bot` precede `redbot.core.commands.context`?
```suggestion
from redbot.core import checks, commands
from redbot.core.bot import Red
from redbot.core.commands.context import Context
from redbot.core.utils.chat_formatting import info, success
```